pkg/cmd/cli/plugin/helpers.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2017 the Velero contributors.
```

Comment:
According to the recently change, the year 2017 should be removed

Reply:
Done.